malaria-ios/Achievement.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Achievement.swift
+//  malaria-ios
+//
+//  Created by Teodor Ciuraru on 7/13/16.
+//  Copyright © 2016 Bruno Henriques. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+
+class Achievement: NSManagedObject {
+
+// Insert code here to add functionality to your managed object subclass
```

Comment:
After further analysis it seems that this empty class shouldn't exist as well as the AchivementProperty empty class